src/lib/djerba/render/database.py

```diff
@@ -0,0 +1,75 @@
+"""Interface with a CouchDB instance for JSON report documents"""
+
+import logging 
+import configparser 
+import requests
+import json
+import os
+
+import djerba.util.constants as constants
+import djerba.util.ini_fields as ini
+from datetime import datetime
+from djerba.util.logger import logger
+
+class Database(logger):
+    """Class to communicate with CouchDB via the API, eg. using HTTP GET/POST statements"""
+
+    def __init__(self, log_level=logging.WARNING, log_path=None):
+        self.logger = self.get_logger(log_level, __name__, log_path)
+        self.logger.info("Initializing Djerba database object")
+    
+    def Merge(self,dict1,dict2):
+        combined = {**dict1, **dict2}
+        return combined
+
+    """ Upload json to couchdb"""
+    def UploadFile(self,json_path):
+        time = datetime.now()
+        dt_couchDB = time.strftime("%d/%m/%Y %H:%M") 
+
+        with open(json_path) as report:
+            data = json.load(report)
+            config = data.get(constants.SUPPLEMENTARY).get(constants.CONFIG)
+            archive_base = config[ini.SETTINGS][ini.ARCHIVE_BASE]
+            archive_name = config[ini.SETTINGS][ini.ARCHIVE_NAME]           
+            db = config[ini.SETTINGS][ini.ARCHIVE_NAME]
+            base = config[ini.SETTINGS][ini.ARCHIVE_BASE]
+            if base[-1] != '/': 
```

Comment:
Cleaner and safer to join URLs with Python's `urllib.parse` library. See: https://stackoverflow.com/a/8223955